### ISSUE TITLE
[Suggestion] Accessibility considerations

### ISSUE THREAD

**Issue by IsaacGraham** [hex-agon/mastering-mixology#76]
Thanks for your work on the plugin so far.

I'm hoping you might be able to add some options to benefit myself and others with poor vision.
The current state of the minigame interface is that it's stuck in the top left corner with rather small text, and visually indistinct icons and text. I find myself perpetually leaning forward and squinting to see it. Please let me know if these suggestions are outside the plugin's scope. The changes I'd like to see implemented are:

- Allow the interface to be repositioned. For some reason this interface is fixed in place and can't be moved with alt+drag like other things.
- Options for text/icon size and colour. The recipe indication helps a lot with this, but it would be nice to also be able to recolour each of the tool icons, so you can tell which tool is needed by colour rather than shape.

Thanks.

**Comment by raiyni** [comment]
I don't believe we can easily move it, even from runelite without modifying something. For whatever reason, jagex implemented the parent widget with very large bounds that's kind of non standard.

**Comment by TWolf-Dev** [comment]
I second this! I've also had a friend that had this same exact experience. While repositioning the overlay may be an issue, is it still acceptable to add options to modify text size and color?

**Comment by xSnook** [comment]
I have a friend with poor vision that needs scaling ability as well. I can look into getting a PR up too

**Comment by IsaacGraham** [comment]
Hi, I saw BlueSoapTurtle's PR working on getting the scaling added. Looked like it was about ready to be merged and didn't end up happening. Was there a problem with it?
